loky/backend/semaphore_tracker.py

```diff
@@ -73,11 +72,13 @@ def ensure_running(self):
                 # process will out live us, so no need to wait on pid
                 exe = spawn.get_executable()
                 args = [exe] + util._args_from_interpreter_flags()
-                if len(args[-1]) > 100:  # pragma: no cover
-                    warnings.warn(
-                        "args_from_flag returned -R+ -> why??", RuntimeWarning)
```

Comment:
Okay this is solved!!!!
The stock version of python2.7 in travis `2.7.9` have the same bug as in python3.3, where you get args `-RRRR...` instead of `-R`. This can be huge (more than 1000 `R`) and cause a crash. 

As I removed the correction for `python2.7`, the tests crash deterministically _e.g._ [here](https://travis-ci.org/tomMoral/loky/jobs/248392787) and [here](https://travis-ci.org/tomMoral/loky/jobs/248392778).
The randomness in the crash before come from the fact that I checked only the last arguments before and that this could be in other position due to the non deterministic loop on `argv` items.